### DISCUSSION TITLE
Does Skiabitmap support 16bit png?

### DISCUSSION THREAD

**Discussion by zhan2016** [mono/SkiaSharp#2635] [Q&A]
Hello,guys. 
   **Does Skiabitmap support 16bit png? How can we decode a 16 bit png file?**
1. the Default decode result is gray8:
![image](https://github.com/mono/SkiaSharp/assets/20983133/ad0b4caa-b511-4afa-8e2e-e0936c5fd364)
2. if we specify colortype, the result will be null:
![image](https://github.com/mono/SkiaSharp/assets/20983133/a47adb83-5d31-4e96-9fbb-745650829042)
the file info:
![image](https://github.com/mono/SkiaSharp/assets/20983133/a9e9bf6e-35a9-4fb3-9d6c-c57fcba1fa02)

the attach file:
![1](https://github.com/mono/SkiaSharp/assets/20983133/5fbad87d-49b1-43fb-88fb-4a0aa2ee8adb)

**Comment by mikebm** [comment]
Ever get an answer to this?
